docs/rafter/05-01-rafter.md

```diff
@@ -0,0 +1,47 @@
+| **controller-manager.envs.clusterAssetGroup.relistInterval** | Determines time intervals in which the Rafter Controller Manager verifies the ClusterAssetGroup for changes. | `5m` |
+| **controller-manager.envs.assetGroup.relistInterval** | Determines time intervals in which the Rafter Controller Manager verifies the AssetGroup for changes. | `5m` |
+| **controller-manager.envs.clusterBucket.region** | Specifies the regional location of the ClusterBucket in a given cloud storage. Use one of [these](https://github.com/kyma-project/kyma/blob/master/resources/cluster-essentials/templates/bucket.crd.yaml#L52) regions. | `us-east-1` |
+| **controller-manager.envs.bucket.region.value** | Specifies the regional location of the bucket in a given cloud storage. Use one of [these](https://github.com/kyma-project/kyma/blob/master/resources/cluster-essentials/templates/bucket.crd.yaml#L52) regions. | `us-east-1` |
+| **controller-manager.envs.clusterBucket.maxConcurrentReconciles** | Defines the maximum number of cluster bucket concurrent reconciles which will run. | `1` |
+| **controller-manager.envs.bucket.maxConcurrentReconciles** | Defines the maximum number of bucket concurrent reconciles which will run. | `1` |
+| **controller-manager.envs.clusterAsset.maxConcurrentReconciles** | Defines the maximum number of cluster asset concurrent reconciles which will run. | `1` |
+| **controller-manager.envs.asset.maxConcurrentReconciles** | Defines the maximum number of asset concurrent reconciles which will run. | `1` |
+| **controller-manager.envs.store.secretKey** | Defines the secret key. Add the parameter to set your own **secretkey** credentials. | By default, **secretKey** is automatically generated. |
+| **controller-manager.envs.store.accessKey** | Defines the access key. Add the parameter to set your own **accesskey** credentials. | By default, **accessKey** is automatically generated. |
+| **controller-manager.envs.store.uploadWorkers** | Defines the number of workers used in parallel to upload files to the storage bucket. | `10` |
+| **controller-manager.envs.webhooks.validation.workers** | Defines the number of workers used in parallel to validate files. | `10` |
+| **controller-manager.envs.webhooks.mutation.workers** | Defines the number of workers used in parallel to mutate files. | `10` |
+| **upload-service.deployment.replicas** | Defines the number of service replicas. | `1` |
+| **front-matter-service.deployment.replicas** | Defines the number of service replicas. For more details, see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/).| `1` |
+| **asyncapi-service.deployment.replicas** | Defines the number of service replicas. | `1` |
+| **asyncapi-service.envs.verbose** |  If set to `true`, you enable the extended logging mode that records more information on AsyncAPI Service activities than the usual logging mode which registers only errors and warnings. | `true` |
```

Comment:
Please also add two additional parameters about verbose option in `front-matter` and `upload` services.

| Parameter | Description | Default value |
|-----------|-------------|---------------|
| **upload-service.envs.verbose** |  If set to `true`, you enable the extended logging mode that records more information on Upload Service activities than the usual logging mode which registers only errors and warnings. | `true` |
| **front-matter-service.envs.verbose** |  If set to `true`, you enable the extended logging mode that records more information on Front Matter Service activities than the usual logging mode which registers only errors and warnings. | `true` |